core/src/css/core.scss

```diff
@@ -201,3 +201,32 @@ ion-card-header.ion-color .ion-inherit-color {
 .md .menu-content-push {
   box-shadow: $menu-md-box-shadow;
 }
+
```

Comment:
One thing I would appreciate feedback from the Ionic Team from is how we can get away with a lot of these global styles or at least make it easier to maintain.